convnets/logistic_sgd.py

```diff
@@ -428,8 +428,8 @@ def sgd_optimization_mnist(learning_rate=0.13, n_epochs=1000,
         )
         % (best_validation_loss * 100., test_score * 100.)
     )
-    print 'The code run for %d epochs, with %f epochs/sec' % (
-        epoch, 1. * epoch / (end_time - start_time))
+    print('The code run for %d epochs, with %f epochs/sec' % (
+        epoch, 1. * epoch / (end_time - start_time)))
 
     print >> sys.stderr, ('The code ran for %.1fs' % ((end_time - start_time)))
```

Comment:
Nevermind, I was running the notebooks and making sure that nothing crashed, but this file isn't being used by any notebook. You're right, this statement causes a syntax error to be raised in Python 3.